docker-compose/is/docker-compose.yml

```diff
@@ -1,34 +1,53 @@
-version: '2.3'
+version: '3.7'
 services:
-  mysql:
```

Comment:
Wait a second. You can't remove the mysql DB from the mix while specifying that it be used in the deployment.toml (see the [database.identity_db] and [database.shared_db] sections).

Besides that, the mysql DB is what is making this docker-compose a production level product. It looks like this commit is not intended for MASTER but should be on a branch called "Sample-App-Playground" or something along those lines.

Reply:
updated